---
title: "Problem_Set_2-3"
author: "Jonah Lin (30616130)"
date: "March 18, 2018"
output: html_document
---

#### Learning objectives: 
Specific emphasis should be placed on the process used to find the answer. Be as comprehensive as possible e.g. provide URLs for web sources, literature citations, etc.

#### Specific Questions:
* How many prokaryotic divisions have been described and how many have no cultured representatives (microbial dark matter)?


    - 89 Bacteria Phyla
    - 20 Archaeal Phyla
    - Up to 1500 MDM (Microbes that live in shadow biosphere)  
    
    
* How many metagenome sequencing projects are currently available in the public domain and what types of environments are they sourced from?


    - ~110K on EBI
    - Types of environments: Sediments/Soil/Gut/Aquatic/Etc.  
    
    
* What types of on-line resources are available for warehousing and/or analyzing environmental sequence information (provide names, URLS and applications)? 


    Shotgun Metagenomics:
    - Assembly = EULER
    - Binning = S-GCOM
    - Annotation = KEGG
    - Analysis Pipelines = Megan 5
    - InG-/m
    - MG-RAGT
    - NCBI
    
    Marker Gene Metagenomics:
    - Standalone Software = OTUbase
    - Analysis Pipelines = SILVA
    - Databases = Ribosomal Database Project (RDP)
    - Denoising = Amplicon Noise  
    
    
* What is the difference between phylogenetic and functional gene anchors and how can they be used in metagenome analysis?   
    
  
    Phylogenetic:
      - Vertical gene transfer
      - Carry phylogenetic information (Allows tree reconstruction)
      - Taxonomic
      - Ideally single-copy
      
    Functional:
      - More horizontal gene transfer
      - ID specific biogeochemical functions associated with measurable effects
      - Not as useful for phylogeny  
      
      
* What is metagenomic sequence binning? What types of algorithmic approaches are used to produce sequence bins? What are some risks and opportunities associated with using sequence bins for metabolic reconstruction of uncultivated microorganisms?  

  
    Types of Algorithms:
      1) Align sequences to database
      2) Group to each other based on DNA characteristics (GC Content, Codon Usage, etc.)
      
    Risks/Opportunities:
      - Incomplete coverage of genome sequence
      - Contamination from different phylogeny  
      
      
* Is there an alternative to metagenomic shotgun sequencing that can be used to access the metabolic potential of uncultivated microorganisms? What are some risks and opportunities associated with this alternative?

  
    - Single Cell Sequencing
    - Enrichment Culturing
    - Functional Screens (Biochemical, etc.)
    - 3rd Gen Sequencing (Nanopore)
    - FISH  
    
    